CHANGELOG.md

```diff
@@ -61,6 +61,10 @@ BIG documentation improvement and cleanups including (also through other commits
 * Minor improvements to the livebooks for the school (German). This is still WIP
 * Added information about color correction to the 3b livebook
 * Added API documentation for the conversion functions
+* Updated the cheatsheet with coordinator information
+* worked on the coordinator livebook (WIP)
+* Documented the effect options
+* Documented the driver options
 
```

Comment:
A lot of changes are missing that should be added here. It's ok to do it in a separate PR